samples/server/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+
+	"go.opencensus.io/trace"
+	"gocloud.dev/gcp"
+	"gocloud.dev/server"
+	"gocloud.dev/server/sdserver"
+)
+
+type GlobalMonitoredResource struct {
+	projectId string
+}
+
+func (g GlobalMonitoredResource) MonitoredResource() (string, map[string]string) {
+	return "global", map[string]string{"project_id": g.projectId}
+}
+
+func main() {
+	addr := flag.String("listen", ":8080", "HTTP port to listen on")
+
+	ctx := context.Background()
+	credentials, err := gcp.DefaultCredentials(ctx)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+	tokenSource := gcp.CredentialsTokenSource(credentials)
```

Comment:
Thinking: Do we want to keep this as GCP-only or show AWS and Azure somehow?

Reply:
Good question. Making this cross-cloud would involve significantly more code, complicating the example. We could just have two examples, one for AWS and one for GCP (no Azure support yet). Specifically, I'm thinking we could just have these as `Example`s in `_test` files so the tests at least compile them. WDYT?

Reply:
[Scratch the last part, I forgot that `go test ./...` will build the samples too]. We already have examples with server using wire, so samples w/o wire (and maybe later with wire too, for comparison) make more sense and we're more free to add documentation and test w/ real Stackdriver locally, etc.

Reply:
> Making this cross-cloud would involve significantly more code, complicating the example. We could just have two examples, one for AWS and one for GCP (no Azure support yet). Specifically, I'm thinking we could just have these as `Example`s in `_test` files so the tests at least compile them. WDYT?

A reasonable assertion we can make here is that this example works for AWS and GCP (Stackdriver is intentionally cross-cloud), so I think this is fine.

> We already have examples with server using wire, so samples w/o wire (and maybe later with wire too, for comparison) make more sense and we're more free to add documentation and test w/ real Stackdriver locally, etc.

I'm not sure how Wire enters into this.

Reply:
Discussed IRL